discord/commands/core.py

```diff
@@ -968,10 +968,6 @@ def _update_copy(self, kwargs: Dict[str, Any]):
         else:
             return self.copy()
 
-    def _set_cog(self, cog):
```

Comment:
I can't figure out why we're removing this

Reply:
I see but the method is still being used in the code, no?

Reply:
It's inherited by `ApplicationCommand`